index.d.ts

```diff
@@ -55,6 +57,8 @@ export interface OverwriteOptions {
 }
 
 export default class MagicString {
+  indentExclusionRanges: Array<ExclusionRange>;
```

Comment:
It looks to me like this could also be `ExclusionRange | Array<ExclusionRange>`, same as elsewhere. (Sneakily, `clone` uses `indentExclusionRanges.slice()`, which works with either a single range or an array of ranges.) Could you use the more general type here? (I may also be misreading it, but it looks like `MagicString.indent` handles either case.)